README.md

```diff
@@ -162,11 +162,22 @@ $ balena push DebugFleet
 
 ### 2. Initial Diagnosis
 
-{{>"masterclass/debugging/initial-diagnosis"}}
+The balenaCloud Dashboard includes the ability to run diagnostics on
+a device to determine its current condition. This should
+be the first step in attempting to diagnose an issue without having to
+actually access the device via SSH. Ensuring diagnostics and health checks
+are examined first ensures that you have a good idea of the state a
+device is in before SSHing into it, as well as ensuring that the information can
+be accessed later if required (should a device be in a catastrophic state). This
+helps significantly in a support post-mortem should one be required.
 
 #### 2.1 Device Health Checks
 
-This will trigger a set of [health checks](https://www.balena.io/docs/reference/diagnostics/) to run on the device, and you should see the all the checks as `Succeeded` in the Success column. This shows that the device is healthy and there are no obvious faults. 
+To run diagnostics through balenaCloud dashboard, head to the `Diagnostics` tab in the sidebar and click the `Run checks` button to start the tests.
```

Comment:
That's not quite correct. The first thing you see is `Device health checks`, that's where you see the `Run checks` button (which you should do too). For diagnostics, you have to switch to the `Device diagnostics` tab once you're on that page

Reply:
Yeah, I think we mean to say device health checks here, so:

> To run ~~diagnostics~~ health checks through balenaCloud dashboard,...

Reply:
Changed